MultiplayerMod/Multiplayer/State/MultiplayerState.cs

```diff
@@ -1,9 +1,14 @@
 ﻿using System;
 using System.Collections.Generic;
+using MultiplayerMod.Core.Extensions;
 
 namespace MultiplayerMod.Multiplayer.State;
 
 [Serializable]
 public class MultiplayerState {
     public Dictionary<IPlayer, PlayerState> Players = new();
+
+    public void Rebuild() {
```

Comment:
A bit misleading as of now, maybe it's better to reset spawned state in the place? The function is too narrow and tightly related to hard sync logic.